TSG/LCM/How-to-Recreate-Customerconfig.xml-For-Validate.md

```diff
@@ -0,0 +1,17 @@
+# How-to-Recreate-Customerconfig.xml-For-Environment-Validator
+# Description 
+## Do not use this TSG if a deployment action has been started
+This article describes how to bring back the customer configuration if it was removed and not recreated during the environment validator action flow.
+# Symptoms
+## Expected Occurance 
+This issue is expected to happen when certain conditions are met. When a validation call is triggered and fails, and then a second validation call is triggered. You will likely see this issue. This is due to the seconds validation call trying to clean up the state of the first validation call but something is locking a file that needs to be cleaned up in the ECE store. So the script fails to clean up and throws an error before the ECEstore can be recreated.
+## Signature 
+On a re-run of the environment validator action the action is unable to be launched and an error like ```Item 'CustomerConfiguration.xml' not found in store 'Default'. The file path is C:\EceStore\efb61d70-47ed-8f44-5d63-bed6adc0fb0f\0b8ae09e-3e34-bd06-78a3-28c0d0d425ca ```.
```

Comment:
the line is incomplete ie error like <<error>>.... comes up?

Reply:
On a re-run of the environment validator action the action is unable to be launched and an error like ```Item 'CustomerConfiguration.xml' not found in store 'Default'. The file path is C:\EceStore\efb61d70-47ed-8f44-5d63-bed6adc0fb0f\0b8ae09e-3e34-bd06-78a3-28c0d0d425ca ``` should appear in the azure portal or arm template response.